gutenberg/style2.css

```diff
@@ -869,3 +869,114 @@ library .latest h1 {
 		word-break: break-word;
 	}
 }
+
+
+/* CUSTOM CSS FOR STYLE UPGRADE OF BIBREC.html */
+/* Styling for the "read online" link */
+.read_html {
+	display: inline-block;
```

Comment:
Minor: figure out if you want tabs or spaces and stick with one. This is true for the existing stuff above this, but now's a good a time as any to be consistent with new code.

Reply:
I didn't notice that. As I've said, the css files are a real mess. That's a whole 'nother project.